gtfs-netex-test/gtfs_show_map.py

```diff
@@ -38,7 +38,14 @@ def main(gtfs_zip_file,map_file):
         route_stop_coords=[]
         for astop in route_stops:
             x=df_stops[df_stops['stop_id']==str(astop)]
-            route_stop_coords.append([x['stop_lat'].values[0],x['stop_lon'].values[0]])
+            if not(len(x['stop_lat'])==0 or len(x['stop_lon'])==0):  # fix for non-existing coordinates
```

Comment:
Are you sure this works? Are there zero entries in the list if no coordinates exist?